pkg/model/piped.proto

```diff
@@ -74,6 +74,8 @@ message Piped {
     // Launcher uses this value to compare with the currently running version
     // to determine whether Piped should be restarted with another version or not.
     string desired_version = 30;
+    // The flag to determine whether Piped should be restarted with current config.
+    bool need_restart = 31;
```

Comment:
@Szarny How about doing it via `Command` instead of via this flag?
FYI, Command is also being used for other tasks like canceling deployment, syncing a new deployment from the web console.